components/Testimonial.js

```diff
@@ -21,7 +21,7 @@ export default function Testimonial({
         <footer className="mt-6">
           <div className="flex">
             <div className="flex-shrink-0 inline-flex rounded-full border-2 border-white">
-              <img className="h-12 w-12 rounded-full" src={authorAvatar} alt={authorName}/>
+              <img className="h-12 w-12 rounded-full" src={authorAvatar} alt=""/>
```

Comment:
I think you shouldn't remove this alt text here, since your PR specifies the addition of alt texts in the images. And the props passed to this component clearly has `authorName` specified in it.

Reply:
@akshatnema  Regarding this particular change, I would love to pick your brain on [this write-up](https://github.com/asyncapi/website/issues/735#issuecomment-1117620923), especially the second point. To me those images appear to be decorative at best, though I could be wrong. 

My reasoning is that when you look at the testimonial and author sections, there is a text just adjacent to each image. The text contains the name of the author which is exactly the same as the `alt` text. This [article on decorative images](https://www.w3.org/WAI/tutorials/images/decorative/)  on W3.org, suggests that if there is a text adjacent to the image which is describing the image, you can add a null `alt` text to prevent redundancy. 

![image](https://user-images.githubusercontent.com/52580190/167303618-a3bd5ee3-3094-45e7-aaac-0a7b173a5288.png)


For our case, though I haven't tested, I believe a screen reader would announce something like "image, Matt McLarty, Matt McLarty, Global Leader of API Strategy at MuleSoft".  You will notice how redundant the `alt` text of the image element is because of the adjacent text which is exactly the same as the `alt` text. 

Though I could be wrong about this, I believe that was the basis for the change. Let me know what you think.

Reply:
Thanks @nibble0101 for providing great info regarding alt text that appears near decorative images and the screen reader examples. I'm still on the side that there is no loss in providing `AuthorName` to the alt inside decorative images because, in the codebase, it gives information to the readers that the image added is of the author who has given a testimonial. It's better that you first ask any of the CODEOWNERS to review the PR as well as the issue and have their views on it. Since, this PR needs an approval review from any of the maintainers, I think we should handle this part directly to them only.

Reply:
> Since, this PR needs an approval review from any of the maintainers, I think we should handle this part directly to them only.

Absolutely.

Reply:
I agree with @nibble0101. However I wonder if we should add `alt` to this decorative image as `Image of {authorName}` without info about position etc. I know it may still be redundant but it would make more sense than an empty `alt`.

Reply:
Hey all, late to the show what with GSoD madness... 😸  but I need to note that I have the same feedback as @akshatnema in regards to adding `alt` to these testimonial images too.

So @nibble0101, your point is founded on interpreting the testimonial images to be _decorative_ (w3). However, I do not think that is correct because a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/) (again, according to W3) is focused on design/making a website attractive. (i.e. Visual styling such as borders, spacers, and corners) I know it is nuanced and the interpretation could be that adding the author pic makes the website look "better" and more attractive. But since these are author pics and not images made for design-y purposes, that is why I do not consider them to be decorative per the w3 definition. 

Rather, I consider the author images to be [informative images](https://www.w3.org/WAI/tutorials/images/informative/), in that these profile pics are "used to supplement other information" of the testimonial section. 

All this to say, my review take on this is that we should not have empty `alt`s here and that they need to be added.